rotkehlchen/db/cache.py

```diff
@@ -14,15 +13,15 @@ class DBCache(Enum):
     LAST_AUGMENTED_SPAM_ASSETS_DETECT_KEY = 'last_augmented_spam_assets_detect_key'
 
     @classmethod
-    def deserialize(cls: type['DBCache'], value: str) -> 'DBCache':
+    def deserialize(cls: type['DBCache'], value: str) -> 'DBCache | str':
         """The method to deserialize a string to an enum object"""
         try:
             return getattr(cls, value.upper())
-        except AttributeError as e:
-            raise DeserializationError(f'Failed to deserialize {cls.__name__} value {value}') from e  # noqa: E501
+        except AttributeError:
+            return value
```

Comment:
Doesn't this make the deserialization logic useless? What is the porpouse of having it now?